src/main/java/edu/ucsb/cs156/happiercows/entities/Commons.java

```diff
@@ -32,6 +32,15 @@
   private double degradationRate;
   private boolean showLeaderboard;
   private int carryingCapacity;
+  private double priceChange;
+  public void increaseCowPrice() {
+    // Implement the logic to increase the cow price
```

Comment:
Are these meant for a later pull request

Reply:
removed